### ISSUE TITLE
A11y: improve keyboard access for buttons on /new editor view

### ISSUE THREAD

**Issue by lisasy** [forem/forem#10521]
# Problem
When I am in the Write a Post editor view (`/new`) and tab through with my keyboard, I cannot select or access certain buttons:

1. "Add cover image": I can navigate to the button, but pressing on it twice doesn't open anything.
2. "Upload image": Tabbing skips this button.
3. "Close" icon in upper-right: Tabbing skips this button.

# Proposed solution

Make all 3 buttons selectable and navigable by tabbing with a keyboard.

**Comment by Manaswini1832** [comment]
I'd like to work on this issue. May I take it up?

**Comment by rhymes** [comment]
Yes @Manaswini1832, thanks! I've assigned this to you
